core/vm/opcodes.go

```diff
@@ -101,6 +101,8 @@ const (
 	NUMBER
 	DIFFICULTY
 	GASLIMIT
+	CHAINID     = 0x46
```

Comment:
Maybe "= 0x46" and "= 0x47" could be omitted because of iota effect.

Reply:
They could, but I dislike the iota effect -- imo it's wrong to use iota to define something that is defined _externally_, and it should only be used for things where there's no external dependency -- where only internal consistency matters